paragon/_fonts.scss

```
@@ -1 +1,29 @@
 // Add any font imports or definitions here
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:600,400,300);
+
+@font-face {
+    font-family: "Avenir";
+    src: url(fonts/avenir-book.woff2);
+    font-weight: 400;
```

Maybe we should specify the "light" weight here too, since we don't have a more appropriate file for that?

```suggestion
    font-weight: 300 400;
```